checker/tests/test_path_generator.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+import AnDO_engine as andoE
+
+import os
+path=os.getcwd()
+
+
+class test_path_generator(unittest.TestCase):
```

Comment:
can you document please? (the class, the functions)

Reply:
Done in https://github.com/INT-NIT/AnDOChecker/pull/17/commits/c13fb706bc8c6d94bbaf2b8581859fc0c212a76f.

Reply:
comments are in the commit!